File: spec/php/PositionInSeqTest.php
<?php
namespace Kaitai\Struct\Tests;

class PositionInSeqTest extends TestCase {
    public function testPositionInSeq() {
        $r = PositionInSeq::fromFile(self::SRC_DIR_PATH . "/position_in_seq.bin");

        $this->assertEquals([1, 2, 3], $r->numbers());
    }
}

File: spec/go/position_in_seq_test.go
package spec

import (
	"runtime/debug"
	"os"
	"testing"
	"github.com/stretchr/testify/assert"
	"github.com/kaitai-io/kaitai_struct_go_runtime/kaitai"
	. "test_formats"
)

func TestPositionInSeq(t *testing.T) {
	defer func() {
		if r := recover(); r != nil {
			debug.PrintStack()
			t.Fatal("unexpected panic:", r)
		}
	}()
	f, err := os.Open("../../src/position_in_seq.bin")
	if err != nil {
		t.Fatal(err)
	}
	s := kaitai.NewStream(f)
	var r PositionInSeq
	err = r.Read(s, &r, &r)
	if err != nil {
		t.Fatal(err)
	}

	assert.EqualValues(t, []uint8{1, 2, 3}, r.Numbers)
}
